frontend/src/app/pages/exercises/exercise/shared/exercise-map/feature-managers/element-manager.ts

```diff
@@ -19,11 +19,17 @@ export abstract class ElementManager<
         UnsupportedChangeProperties
     > = Exclude<ReadonlySet<keyof Element>, UnsupportedChangeProperties>
 > {
+    /**
+     * This type is used to identify the type of an element, e.g., when it was (drag &) dropped on something
```

Comment:
"make it more descriptive" is not a good review comment. In such cases, you should try to come up with your own guess/proposal of what should be there because I now can only guess what exactly wasn't clear to you...

Keep in mind that this will probably be replaced in the near future with a `type` property in each element.

```
When an element gets (dragged &) dropped, this identifies the type of the dropped element.
@example `materials`
```